### ISSUE TITLE
[A11Y][Text input] Bottom border too low for text input

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#745]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

Samsung S25

### OS Version

Android 16

### App version

1.3.0

### Describe the issue

Bottom border contrast on text input is not high enough:

- on white background, outline is grey #C7C7C7 which leads to a 1.7:1 contrast ratio instead of 3:1 minimum (on native Android app, the border is correct #575757)

<img width="655" height="165" alt="" src="https://github.com/user-attachments/assets/36b03e5c-4c58-4398-9496-4d8573075f68" />

- on dark background, outline is grey #454545 which leads to a 1.9:1 contrast ratio instead of 3:1 minimum (on native app, color is around #ABABAB)

### Expected Behavior

Bottom border should have a contrast of 3:1 minimum
Use the same color tokens than native Android app

<img width="718" height="376" alt="" src="https://github.com/user-attachments/assets/5daabee1-267f-481e-8086-e30c4af51fc6" />

**Comment by pya35** [comment]
do the same on text area

**Comment by nouha06** [comment]
@pya35 according to the [figma](https://www.figma.com/design/ZEyIRs5X9uPfUbmlglCVHS/Control---Text-input?node-id=7448-21039&m=dev) design, we used the correct token _ouds/💠_control/text-input/color/border/enabled_  when the status is enabled. So in this case, I think we should ask the design team about that.
cc @MaximeTonnerre — and we should also check with the Android team which token they are using for this. cc @paulinea

**Comment by pya35** [comment]
@nouha06 @MaximeTonnerre same for outlined mode: borders shall reach 3:1 contrast ratio 
see also [iOS issue]( https://github.com/Orange-OpenSource/ouds-ios/issues/1058) and [Android issue](https://github.com/Orange-OpenSource/ouds-android/issues/1298)

**Comment by paulinea** [comment]
> [@pya35](https://github.com/pya35) according to the [figma](https://www.figma.com/design/ZEyIRs5X9uPfUbmlglCVHS/Control---Text-input?node-id=7448-21039&m=dev) design, we used the correct token _ouds/💠_control/text-input/color/border/enabled_ when the status is enabled. So in this case, I think we should ask the design team about that. cc [@MaximeTonnerre](https://github.com/MaximeTonnerre) — and we should also check with the Android team which token they are using for this. cc [@paulinea](https://github.com/paulinea)

You're right @nouha06, this one is a design issue. The design needs to evolve so that we can take it into account on our end.

**Comment by pya35** [comment]
@MaximeTonnerre @nouha06 

2 issues here:
- bottom border should be contrasted, as done on Android native (check token used on Android for this case)
- outlined mode (4 borders) should be constrated for all apps

**Comment by nouha06** [comment]
Maxime response : 
> _we had already identified this issue several weeks/months ago. It is already in our todo list and will be fixed soon_

We will fix it once the design is provided.
